<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/resourcen.qrc">
    <normaloff>:/icons/gtk-preferences.png</normaloff>:/icons/gtk-preferences.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="settings_tab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <widget class="QWidget" name="keyboard_tab">
      <attribute name="icon">
       <iconset resource="icons/resourcen.qrc">
        <normaloff>:/icons/keyboard.png</normaloff>:/icons/keyboard.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Keyboard</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="action_list">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>4</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>150</number>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Shortcut</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Shortcut</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="shortcut"/>
         </item>
         <item>
          <widget class="QPushButton" name="clear_button">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
           <property name="icon">
            <iconset resource="icons/resourcen.qrc">
             <normaloff>:/icons/dialog-error.png</normaloff>:/icons/dialog-error.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reset_button">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Reset</string>
           </property>
           <property name="icon">
            <iconset resource="icons/resourcen.qrc">
             <normaloff>:/icons/gtk-undo-ltr.png</normaloff>:/icons/gtk-undo-ltr.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="default_button">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Default</string>
           </property>
           <property name="icon">
            <iconset resource="icons/resourcen.qrc">
             <normaloff>:/icons/go-jump.png</normaloff>:/icons/go-jump.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mouse_tab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="icon">
       <iconset resource="icons/resourcen.qrc">
        <normaloff>:/icons/mouse.png</normaloff>:/icons/mouse.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Mouse</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Roate x</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Clear</string>
         </property>
         <property name="icon">
          <iconset resource="icons/resourcen.qrc">
           <normaloff>:/icons/dialog-error.png</normaloff>:/icons/dialog-error.png</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Wheel zoom x10 button</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="rotate_x_key"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="rotate_y_key"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="scale_key"/>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>Clear</string>
         </property>
         <property name="icon">
          <iconset resource="icons/resourcen.qrc">
           <normaloff>:/icons/dialog-error.png</normaloff>:/icons/dialog-error.png</iconset>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="rotate_z_key"/>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Clear</string>
         </property>
         <property name="icon">
          <iconset resource="icons/resourcen.qrc">
           <normaloff>:/icons/dialog-error.png</normaloff>:/icons/dialog-error.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Roate y</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Roate z</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>Clear</string>
         </property>
         <property name="icon">
          <iconset resource="icons/resourcen.qrc">
           <normaloff>:/icons/dialog-error.png</normaloff>:/icons/dialog-error.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Click button</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="swap_wheel_zoom">
         <property name="text">
          <string>Swap wheel zoom</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="click_button"/>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="wheel_zoom_x10">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="18">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>189</width>
           <height>178</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="15" column="0" rowspan="3" colspan="3">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>148</width>
           <height>133</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gui_tab">
      <attribute name="icon">
       <iconset resource="icons/resourcen.qrc">
        <normaloff>:/icons/gui.png</normaloff>:/icons/gui.png</iconset>
      </attribute>
      <attribute name="title">
       <string>GUI</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="invert_z_rotation">
         <property name="text">
          <string>Invert z rotation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Toolbar icon size</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="icon_size_small">
            <property name="text">
             <string>Small (16x16)</string>
            </property>
            <property name="icon">
             <iconset resource="icons/resourcen.qrc">
              <normaloff>:/icons/gtk-execute.png</normaloff>:/icons/gtk-execute.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="icon_size_medium">
            <property name="text">
             <string>Medium (24x24)</string>
            </property>
            <property name="icon">
             <iconset resource="icons/resourcen.qrc">
              <normaloff>:/icons/gtk-execute.png</normaloff>:/icons/gtk-execute.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="icon_size_big">
            <property name="text">
             <string>Big (32x32)</string>
            </property>
            <property name="icon">
             <iconset resource="icons/resourcen.qrc">
              <normaloff>:/icons/gtk-execute.png</normaloff>:/icons/gtk-execute.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="icon_size_verry_big">
            <property name="text">
             <string>Verry big (48x48)</string>
            </property>
            <property name="icon">
             <iconset resource="icons/resourcen.qrc">
              <normaloff>:/icons/gtk-execute.png</normaloff>:/icons/gtk-execute.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="dirs_tab">
      <attribute name="icon">
       <iconset resource="icons/resourcen.qrc">
        <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Dirs</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>EL data dir</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="el_data_dir_button">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>EL2 data dir</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="el2_data_dir_button">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>158</width>
           <height>145</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="textures_tab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="icon">
       <iconset resource="icons/resourcen.qrc">
        <normaloff>:/icons/terrain.png</normaloff>:/icons/terrain.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Textures</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QListView" name="textures"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="textures_clear">
           <property name="text">
            <string>Clear</string>
           </property>
           <property name="icon">
            <iconset resource="icons/resourcen.qrc">
             <normaloff>:/icons/dialog-error.png</normaloff>:/icons/dialog-error.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="textures_add">
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="icons/resourcen.qrc">
             <normaloff>:/icons/gtk-add.png</normaloff>:/icons/gtk-add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="textures_remove">
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="icon">
            <iconset resource="icons/resourcen.qrc">
             <normaloff>:/icons/editclear.png</normaloff>:/icons/editclear.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="icons/resourcen.qrc">
         <normaloff>:/icons/gtk-cancel.png</normaloff>:/icons/gtk-cancel.png</iconset>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="icon">
        <iconset resource="icons/resourcen.qrc">
         <normaloff>:/icons/gtk-apply.png</normaloff>:/icons/gtk-apply.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="import_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Import</string>
       </property>
       <property name="icon">
        <iconset resource="icons/resourcen.qrc">
         <normaloff>:/icons/fileopen.png</normaloff>:/icons/fileopen.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
       <property name="icon">
        <iconset resource="icons/resourcen.qrc">
         <normaloff>:/icons/filesave.png</normaloff>:/icons/filesave.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons/resourcen.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ok_button</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settings_tab</sender>
   <signal>currentChanged(int)</signal>
   <receiver>shortcut</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
